setup.py

```diff
@@ -12,6 +12,10 @@
 
 ext_kwds = dict(include_dirs=include_dirs(), library_dirs=library_dirs())
 
+# Generate auto-generated sources from pari.desc
+# This needs to be done before build/build_ext so the generated pxd is moved
+# to the build directory and installed with newer setuptools.
+rebuild()
```

Comment:
This will make this run even on just building the sdist, which does not seem correct

Reply:
I had not thought of that because I did not produce sdist off course. It may need to be in `build`.

Reply:
I just did a basic test by adding a build class and running `rebuild` there. The `.pxd` are only generated once but are now properly shipped with each python implementation. I would prefer if `.pxd` where always re-generated but that may be considered an efficiency win. I will check what happens to sdist.

Reply:
No, that still generate and incorporate `.pxd` in sdist. I may have to use `MANIFEST.in` instead to exclude them.

Reply:
I keep forgetting. This is not about all `.pxd` files, it is about just `auto_paridecl.pxd`. The sdist is fine putting `rebuild` in a `build` class.

Reply:
And `auto_paridecl.pxd` is also not included in sdist with my current PR.